### DISCUSSION TITLE
在本地创建的服务器使用云服务器的数据库内容进行解析展示web内容

### DISCUSSION THREAD

**Discussion by yutonghome** [plan-player-analytics/Plan#3724] [Q&A] [answered]
我在云端拥有一个mc服务器 但无法通过控制台的链接直接访问   我想通过本地搭建相同版本的服务器   解析云端数据库中的数据进行展示  我尝试直接将云端mysql的数据  备份到本地mysql  然后修改config.yml  解析本地mysql的数据 可是每次启动本地服务器  数据貌似会被清空 不会正确展现已有的mysql数据

**Comment by yutonghome** [comment]
具体的是  我想使用旧的mysql数据  而不加载最新的数据

**Comment by AuroraLS3** [comment, marked as answer]
I had to use screen translator so this answer might not be what you're looking for

- Create a 2nd mysql database (`CREATE DATABASE...` (see wiki)) 
- mysql dump the 1st
- import to 2nd database
- Connect a Plan instance to 2nd database, copy plugins/Plan/ServerInfoFile.yml from 1st instance so that it reads the data of the same server

**Reply by yutonghome** [reply]
My foreign language is average, and using a translator to translate the following content may result in inaccurate expression.
Thank you for your reply! Sorry, this is my first time seeking help with a problem on a similar website, and I feel that my description format is a bit inappropriate.
After trying the method you proposed, it was indeed resolved.
And in the temporary viewing scenario, I found that using SQLite database to copy the entire folder of Plan from the cloud server and start it in the local server folder can accurately display the content at the time of copying.

**Reply by yutonghome** [reply]
Thank you again for your reply
